Source/gs/GSH_OpenGL/GSH_OpenGL.cpp

```diff
@@ -271,6 +297,12 @@ void CGSH_OpenGL::FlipImpl()
 
 	PresentBackbuffer();
 	CGSHandler::FlipImpl();
+
+	if(framebuffer)
+	{
+		std::swap(framebuffer->m_texture, framebuffer->m_texture2);
```

Comment:
I don't think this works since most games use double buffering. You will end up with different framebuffer pointers every frame. You could try keeping the last framebuffer used if you'd like to do a quick test, but I don't think it would be a reliable method in the long run.